lib/google_scraping/account/helpers/user_token.ex

```diff
@@ -0,0 +1,39 @@
+defmodule GoogleScraping.Accounts.UserToken do
+  use Ecto.Schema
+  alias GoogleScraping.Accounts.UserToken
+
```

Comment:
Lets add a new line for each type as per our convention https://nimblehq.co/compass/development/code-conventions/elixir/#modules
```suggestion
  use Ecto.Schema
  
  import Ecto.Query
  
  alias GoogleScraping.Accounts.UserToken
```